yarn.lock

```diff
@@ -5255,7 +5255,7 @@ first-mate@^7.0.2:
     event-kit "^2.2.0"
     fs-plus "^3.0.0"
     grim "^2.0.1"
-    oniguruma "7.2.1"
+    oniguruma "7.2.3"
```

Comment:
This was fixed in https://github.com/palantir/blueprint/pull/5407. Just merge in develop